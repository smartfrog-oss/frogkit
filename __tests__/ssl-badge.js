import { mount } from 'vue-test-utils'
import SslBadge from '@/components/ssl-badge'

describe('Ssl Badge component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(SslBadge, {propsData: {text: 'Some long text goes here'}})
    expect(wrapper.html()).toMatchSnapshot()
  })

})
