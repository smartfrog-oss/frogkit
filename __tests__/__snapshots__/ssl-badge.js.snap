// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ssl Badge component should render component and match snapshot 1`] = `
<flex align="center" class="fk-ssl-badge"><img src="../assets/images/ssl.png" class="fk-ssl-badge__img">
  <div class="fk-ssl-badge__text">Some long text goes here</div>
</flex>
`;
